src/psyclone/tests/psyir/transformations/transformations_test.py

```diff
@@ -407,10 +407,6 @@ def create_loops(body):
     # just with 'jk' and it is not modified in the inner loops
     omplooptrans.validate(loops[1])
 
-    # Check if there is missing symbol information it still validates
```

Comment:
This test would indeed fail now (since the `try` was removed from the parallel_loop_trans). Just to verify, we indeed do not need this anymore? I.e. for all our current applications we have the required information from the symbol table?

Reply:
That should be the case. Certainly, the integration tests are all fine.